package-lock.json

```diff
@@ -3853,11 +3853,11 @@
       "integrity": "sha512-NmWvPnx0F1SfrQbYwOi7OeaNGokp9XhzNioJ/CSBs8Qa4vxug81mhJEAVZwxXuBmYB5KDRfMq/F3RR0BIU7sWg=="
     },
     "node_modules/axios": {
-      "version": "1.6.2",
-      "resolved": "https://registry.npmjs.org/axios/-/axios-1.6.2.tgz",
-      "integrity": "sha512-7i24Ri4pmDRfJTR7LDBhsOTtcm+9kjX5WiY1X3wIisx6G9So3pfMkEiU7emUBe46oceVImccTEM3k6C5dbVW8A==",
+      "version": "1.7.4",
```

Comment:
Shouldn't we be updating the deps in `package.json` ?

Reply:
Not always the case, if you wanna bump some "sub" deps you don't have in `package.json`. Please refer to the "Closed" PR (#241 , #244 , #245 , #246 , #247) to match the changes